core/app/models/spree/product.rb

```diff
@@ -94,8 +91,6 @@ def find_or_build_master
     validates :shipping_category_id, presence: true
     validates :slug, length: { minimum: 3 }, uniqueness: { allow_blank: true }
 
-    attr_accessor :option_values_hash
```

Comment:
I don't think this should be removed, it's a parameter accepted through the API.